File: Codigo/SistemaVeterinario/logica/persona.cs
namespace SistemaVeterinario.logica
{
    public class Persona
    {
        public string strNombre { get; set; }
        public string strTelefono { get; set; }

        public Persona(string strNombre, string strTelefono)
        {
            this.strNombre = strNombre;
            this.strTelefono = strTelefono;
        }

        public void mostrarInformacion()
        {
            Console.WriteLine("Nombre: " + strNombre);
            Console.WriteLine("Telefono: " + strTelefono);
        }
    }
}

File: BlueJ/DiagramaDeClases/Persona.java
public class Persona
{
    String strNombre;
    String strTelefono;

    public Persona(String strNombre, String strTelefono)
    {
        this.strNombre = strNombre;
        this.strTelefono = strTelefono;
    }

    public void mostrarInformacion()
    {
        System.out.println("Nombre: " + strNombre);
        System.out.println("Telefono: " + strTelefono);
    }
}